FlowCrypt/Controllers/Settings/KeySettings/Key List/KeySettingsViewController.swift

```diff
@@ -44,9 +44,14 @@ final class KeySettingsViewController: TableNodeViewController {
         node.delegate = self
         node.dataSource = self
         node.reloadData()
-
-        loadKeysFromStorageAndRender()
         setupNavigationBar()
+        Task {
+            do {
+                try await loadKeysFromStorageAndRender()
+            } catch {
+                handleCommon(error: error)
+            }
+        }
```

Comment:
Maybe we could have a convention for this. Like maybe call it `loadKeysTask` or `somethingElseTask` if it's a method that creates a task for async methods. Then, by looking at the method name, you know what type of method it is.

Generally, there should be not more than one Task method per such handler (except when we want things to happen in parallel), so such convention would allow us to see the handler at a glance.

Reply:
Sounds good 👍